contracts/src/plugin/Plugin.sol

```diff
@@ -72,7 +85,10 @@ abstract contract Plugin is IPlugin, ERC165, DaoAuthorizable, ProtocolVersion {
         return
             _interfaceId == type(IPlugin).interfaceId ||
             _interfaceId == type(IProtocolVersion).interfaceId ||
-            _interfaceId == this.setTargetConfig.selector ^ this.getTargetConfig.selector ||
+            _interfaceId ==
+            this.setTargetConfig.selector ^
+                this.getTargetConfig.selector ^
+                this.getCurrentTargetConfig.selector ||
```

Comment:
do we need to add both `getTargetConfig` and `getCurrentTargetConfig` to the interface?

Reply:
Why not ?